validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -271,14 +271,21 @@ public static ValidatorClientService create(
             })
         .exceptionally(
             error -> {
-              final Optional<Throwable> maybeCause =
-                  ExceptionUtil.getCause(error, InvalidConfigurationException.class);
-              if (maybeCause.isPresent()) {
-                LOG.warn(maybeCause.get().getMessage());
-              } else {
-                LOG.error("Error was encountered during validator client service start up.", error);
-              }
-              checkNoKeysLoaded(validatorConfig, validatorLoader);
```

Comment:
`checkNoKeysloaded` here was the core problem of the ticket leaving us parked.
In this case we need to system.exit, because some or all of the chain above has failed, and the last item in the chain is to perform duties, so we're literally parked with no duties.

Reply:
I agree that this option should only be considered when everything (keys loading) run without exception only